rust/scx_utils/src/topology.rs

```diff
@@ -139,6 +141,8 @@ pub struct Core {
     /// Ancestor IDs.
     pub llc_id: usize,
     pub node_id: usize,
+    /// The sysfs value of core_id
```

Comment:
I'd put this right below `id` and explain why there are two separate IDs.

Reply:
Let's move this up right below `.id`.

Reply:
I think with this way it gets included in the rust doc though. Or did you mean the location and not the comment... sorry I misinterpreted...

Reply:
Oh yeah, the location of the field as `.id` and `.kernel_id` are closely related, I think it'd be better to colocate them.